samples/snippets/subscriber.py

```diff
@@ -946,6 +984,13 @@ def callback(message: pubsub_v1.subscriber.message.Message) -> None:
     create_push_parser.add_argument("subscription_id")
     create_push_parser.add_argument("endpoint")
 
+    create_push_no_wrapper_parser = subparsers.add_parser(
+        "create-push-no-wrapper", help=create_push_no_wrapper_subscription.__doc__
+    )
+    create_push_parser.add_argument("topic_id")
```

Comment:
Should it be create_push_no_wrapper_parser?

Reply:
Done